XP-Pen_Artist16_Pro/index.md

```diff
@@ -0,0 +1,19 @@
+---
+VID: "28bd"
+PID: "900a"
+vendor: XP-Pen
+product: Artist16 Pro
```

Comment:
The official name has space between "Artist" and "16", so we should have that too, everywhere.